manifests/g/Google/PlatformTools/35.0.2/Google.PlatformTools.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: Google.PlatformTools
+PackageVersion: 35.0.2
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/
+PublisherSupportUrl: https://developer.android.com/tools#tools-platform
+PrivacyUrl: https://policies.google.com/privacy
+PackageName: Android SDK Platform-Tools

```

```suggestion
PackageName: Android SDK Platform Tools
```

Google uses "SDK Platform Tools" on their site. No dash.